pkg/debugapi/peer_test.go

```diff
@@ -56,14 +57,14 @@ func TestConnect(t *testing.T) {
 }
 
 func TestDisconnect(t *testing.T) {
-	address := "985732527402"
-	unknwonAdddress := "123456"
+	address := "ca1e9f3938cc1425c6061b96ad9eb93e134dfe8734ad490164ef20af9d1cf59c"
```

Comment:
Use swarm.Address for address variables and Equal in mock disconnect func (removing switch).